airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/util/Jsons.kt

```diff
@@ -19,13 +20,19 @@ import java.math.BigInteger
 import java.nio.ByteBuffer
 
 object Jsons : ObjectMapper() {
+    // allow jackson to deserialize anything under 100 MiB
+    // (the default, at time of writing 2024-05-29, with jackson 2.15.2, is 20 MiB)
+    private const val JSON_MAX_LENGTH = 100 * 1024 * 1024
```

Comment:
this diff is just matching our existing behavior from https://github.com/airbytehq/airbyte/blob/712ea82d050fd731b3289697bc56c214a3a33955/airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/json/Jsons.kt#L34

(it's also needed to make the float-handling test work)